### ISSUE TITLE
RPM signature check is failing

### ISSUE THREAD

**Issue by LaszloHajduTesco** [osquery/osquery#8183]
# RPM signature check is failing

### What operating system and version are you using?

Operating system is Centos7

### What version of osquery are you using?

Osquery version: 5.10.2
Rpm version: 4.11.3

### What steps did you take to reproduce the issue?

curl -k https://pkg.osquery.io/rpm/osquery-5.10.2-1.linux.x86_64.rpm --output osquery.pem
curl -L https://pkg.osquery.io/rpm/GPG | sudo tee /etc/pki/rpm-gpg/RPM-GPG-KEY-osquery
yum install rpm
rpm --checksig -v osquery.pem

### What did you expect to see?
osquery.rpm:
    Header V4 RSA/SHA512 Signature, key ID c9d8b80b: OK
    Header SHA1 digest: OK (24db73b6191a65e799680a6928ecef5eed73329a)
    MD5 digest: OK

### What did you see instead?
osquery.rpm:
    Header V4 RSA/SHA512 Signature, key ID c9d8b80b: OK
    Header SHA1 digest: OK (24db73b6191a65e799680a6928ecef5eed73329a)
    MD5 digest: BAD Expected(07057a379c710c2701d47c931479185a) != (3b304d19e2f76c4ee1c5ac4b49273614)

**Comment by Smjert** [comment]
@LaszloHajduTesco you might want to check the SHA256 of the package with `sha256sum osquery-5.10.2-1.linux.x86_64.rpm` first, and compare it with the one displayed in this page: https://osquery.io/downloads/official/5.10.2, which should be `61ef2351a07dbc36ae9ebff605e8a7ecc4e09a07ac11f540d2aed78c143addbe`.

I downloaded the same package and I have the correct MD5 with RPM and SHA256. I think the package got corrupted when downloaded.

**Comment by LaszloHajduTesco** [comment]
I downloaded it again from the official webpage, and at this time I'm getting another result.

1. curl -L https://pkg.osquery.io/rpm/GPG | tee /etc/pki/rpm-gpg/RPM-GPG-KEY-osquery
2. yum install rpm
3. sudo yum-config-manager --add-repo https://pkg.osquery.io/rpm/osquery-s3-rpm.repo
4. repotrack -p /repos osquery-5.10.2-1.linux
5. rpm --checksig -v repos/osquery-5.10.2-1.linux.aarch64.rpm

And the result is:
repos/osquery-5.10.2-1.linux.aarch64.rpm: RSA sha1 md5 NOT OK
With verbose mode:
repos/osquery-5.10.2-1.linux.aarch64.rpm:
    Header V4 RSA/SHA512 Signature, key ID c9d8b80b: NOKEY
    Header SHA1 digest: OK (92bd6915181289148bda12bce14ca54843899a64)
    MD5 digest: OK (254c3e965c56210630a878f432b34867)
   
The result is the same after I downloaded the rpm files (x86_64, aarch64) from the official webpage.
